app/code/Magento/Theme/Block/Html/Footer.php

```diff
@@ -20,6 +20,13 @@ class Footer extends \Magento\Framework\View\Element\Template implements \Magent
     protected $_copyright;
 
     /**
+     * Miscellaneous HTML information
+     *
+     * @var string
+     */
+    protected $_misc;
```

Comment:
Please declare new property as `private` if you don't have strong reasons to use `protected`.
As Magento 2 follows  [PSR-2](http://www.php-fig.org/psr/psr-2/) please do not use single underscores in **new** property names.
For more information see [Magento PHP Coding Standarts](http://devdocs.magento.com/guides/v2.0/coding-standards/code-standard-php.html)

Name of property is to general. Please use more descriptive names (e.g. $miscellaneousHtml)